### DISCUSSION TITLE
how can I add a new routes ?

### DISCUSSION THREAD

**Discussion by Everton617** [boxyhq/saas-starter-kit#1421] [Q&A]
How could I add new routes? I'm new to using NextJs, and I thought that to create new routes I would just need to create a new folder within the "pages" directory, but apparently that's not how it works in the project

**Comment by deepakprabhakara** [comment]
That's right, you add it to pages and then include it in `TeamNavigation.tsx` so it's added to the sidebar.

**Reply by Everton617** [reply]
I'm trying but don't work... I created a new folder whithin the "pages" and include it in TeamNavigation.tsx 

`import { Cog6ToothIcon, CodeBracketIcon } from '@heroicons/react/24/outline';
import { useTranslation } from 'next-i18next';
import NavigationItems from './NavigationItems';
import { NavigationProps, MenuItem } from './NavigationItems';

interface NavigationItemsProps extends NavigationProps {
  slug: string;
}

const TeamNavigation = ({ slug, activePathname }: NavigationItemsProps) => {
  const { t } = useTranslation('common');

  const menus: MenuItem[] = [
    {
      name: t('all-products'),
      href: `/teams/${slug}/products`,
      icon: CodeBracketIcon,
      active: activePathname === `/teams/${slug}/products`,
    },
    {
      name: t('all-test'),
      href: `/test`,
      icon: CodeBracketIcon,
      active: activePathname === `/test`,
    },
    {
      name: t('settings'),
      href: `/teams/${slug}/settings`,
      icon: Cog6ToothIcon,
      active:
        activePathname?.startsWith(`/teams/${slug}`) &&
        !activePathname.includes('products'),
    },
  ];

  return <NavigationItems menus={menus} />;
};

export default TeamNavigation;
`
